November/Palidrome.js

```diff
@@ -0,0 +1,19 @@
+// [November]/[Day 4]/[Javascript]/{Palidrome.js}
```

Comment:
Let this be in a different branch, says this can be ```issue-01-palindrome`` as the branch name. and your push the branch.
```bash
git push <remote> <branch name>
```